index.html

```diff
@@ -24,13 +24,11 @@
             <h1>Hey There.
                 I’m Michael</h1>
         </section>
-        <section class="intro-title">
```

Comment:
Please follow the design you have selected from the design page. The idea of this project is to copy the Figma design as it is and use the font, color, spacing, and images from the design page. Kindly see the screenshot below of the project and make changes so that it is the same as the design page.
![design](https://user-images.githubusercontent.com/35504149/133643848-90456975-56ca-4c9e-b45f-bca17a9c69c8.png)